corelib-definitions/src/main/java/eu/europeana/corelib/definitions/model/facets/logic/CommonTagExtractor.java

```diff
@@ -0,0 +1,816 @@
+package eu.europeana.corelib.definitions.model.facets.logic;
```

Comment:
Although I do understand why, I'm not really happy to see the same list defined twice. The Guava library has a HashBiMap where the map can be used in two directions.
Please use this instead of the classes in reverselogic package....